tests/test_docs/helper.py

```diff
@@ -139,6 +144,8 @@ def _python_selector(cls, block: Dict) -> bool:
 class BasePythonMarkdownDocs(BaseTestMarkdownDocs):
     """Test Markdown documentation by running Python snippets in sequence."""
 
+    REPLACE_VENDOR_TO_PACKAGES = False
```

Comment:
This indicates whether or not we need to replace "from vendor" to "from packages" to make the tests pass. I don't like this approach so much but I haven't figured out a better way for now.